lineapy/api/api.py

```diff
@@ -373,6 +373,7 @@ def to_pipeline(
     reuse_pre_computed_artifacts: List[str] = [],
     generate_test: bool = False,
     pipeline_dag_config: Optional[AirflowDagConfig] = {},
+    include_non_slice_as_comment: bool = False,
```

Comment:
This should not appear in diffs anymore. maybe rebase the branches involved to remove changes you didnt add.